nsscache.conf

```diff
@@ -146,6 +146,11 @@ files_cache_filename_suffix = cache
 
 ldap_base = ou=group,dc=example,dc=com
 ldap_filter = (objectclass=posixGroup)
+# If ldap_nested_groups is enabled, any groups are members of other groups
```

Comment:
Can you add this also to nsscache.conf.5 please?

Reply:
Added docs to nsscache.conf.5